src/views/auth/signin/Basic.vue

```diff
@@ -193,8 +203,40 @@ export default {
     VsudSwitch,
     VsudButton,
   },
-  data() {
-    return { bgImg };
+  setup() {
+    const router = useRouter();
+    const state = reactive({
+      email: "",
+      password: "",
+    });
+    return {
+      updateEmail(newValue) {
+      state.email = newValue.target.value;
+    },
+    updatePassword(newValue) {
+      state.password = newValue.target.value;
+    },
+    signIn() {
+      if (state.email && state.password) {
+        localStorage.setItem("token", "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJPbmxpbmUgSldUIEJ1aWxkZXIiLCJpYXQiOjE3MTU4NjMyMjksImV4cCI6MTc0NzM5OTIyOSwiYXVkIjoid3d3LmV4YW1wbGUuY29tIiwic3ViIjoianJvY2tldEBleGFtcGxlLmNvbSIsIkdpdmVuTmFtZSI6IkpvaG5ueSIsIlN1cm5hbWUiOiJSb2NrZXQiLCJFbWFpbCI6Impyb2NrZXRAZXhhbXBsZS5jb20iLCJSb2xlIjpbIk1hbmFnZXIiLCJQcm9qZWN0IEFkbWluaXN0cmF0b3IiXX0.OexXQaQsIeYHslj0HGdKUbkGdSUvQc5Zi9CjhkFj9zQ");
```

Comment:
It newer sends it to API.

Reply:
I try to send to /Auth/sign-in but return 404, but in real project my solution will be working, since i implement it also.